edison/app.py

```diff
@@ -0,0 +1,15 @@
+import edison
+
+from flask import render_template
+
+
+# API description in swagger - https://app.swaggerhub.com/apis/DoRTaL94/UserManagment/1.0.0
```

Comment:
is this comment relevant to this PR?

Reply:
No, it would probably be relevent when we will actually add the app routes.
I'll remove it.